WalletWasabi.Tests/IntegrationTests/LiveServerTests.cs

```diff
@@ -160,5 +170,11 @@ public async Task GetLegalDocumentsTestsAsync(NetworkType networkType)
 		}
 
 		#endregion Wasabi
+
+		private TorHttpClient MakeTorHttpClient(NetworkType networkType)
+		{
+			Uri baseUri = LiveServerTestsFixture.UriMappings[networkType];
+			return new TorHttpClient(baseUri, Common.TorSocks5Endpoint, isolateStream: true);
```

Comment:
How about setting isolateStream false here. This will change the behavior but it is similar to what we are doing in production. All of these requests are using the same stream.

Reply:
I would say it's up to you and @nopara73 to decide that.

Reply:
You can use false.

Reply:
Changed.